src/ng-select/tsconfig.lib.json

```diff
@@ -17,7 +17,8 @@
     "strictMetadataEmit": true,
     "fullTemplateTypeCheck": true,
     "strictInjectionParameters": true,
-    "enableResourceInlining": true
+    "enableResourceInlining": true,
+    "enableIvy": false
```

Comment:
wonder why Ivy is not supported

Reply:
Not sure - again, just a straight merge from the official - can play with things as much as we want of course but I definitely wasn't going to :)